server/graphql/core/src/loader/patient.rs

```diff
@@ -0,0 +1,68 @@
+use actix_web::web::Data;
+use async_graphql::dataloader::*;
+use repository::EqualFilter;
+use service::{
+    programs::patient::{Patient, PatientFilter},
+    service_provider::ServiceProvider,
+};
+use std::collections::HashMap;
+
+use crate::standard_graphql_error::StandardGraphqlError;
+
+use super::IdPair;
+
+#[derive(Clone)]
+pub struct EmptyPayload;
+pub type PatientLoaderInput = IdPair<EmptyPayload>;
+impl PatientLoaderInput {
+    pub fn new(store_id: &str, name_id: &str) -> Self {
+        PatientLoaderInput {
+            primary_id: store_id.to_string(),
```

Comment:
Is store_id needed ? Patient query shoulnd't need store id anymore

Reply:
I merged recent changes in

Reply:
I was meaning store_id not needed.

https://github.com/openmsupply/open-msupply/compare/1683-report-printing-graphql-changes...1683-report-printing-graphql-changes(Remove-store-id-from-patients-loader)

Reply:
ok I see, sorry I assumed the stored_id was needed for the PatientNode, but the loader only returns the Patient... I merged your changes in, thanks!